packages/patternlab/styleguide/source/assets/js/modules/skipLinkTarget.js

```diff
@@ -5,31 +5,50 @@
 // For figure content skip links
 document.querySelectorAll(".ma__figure__skip-link").forEach(link => {
 
-  let linkTarget = "";
-
   link.addEventListener("click", (e) => {
     // Find the matched linkTarget.
-    let sibling = e.currentTarget.nextElementSibling;
+    let targetId = e.target.getAttribute("href");
+    let skipTarget = document.querySelector(targetId);
+    let skipTargetAnchor = skipTarget.querySelector("a");
+    skipTarget.style.paddingTop = "80px";
```

Comment:
can we move these styles into a CSS class? That way we can control these styles in one place instead of two.

Reply:
Moved them to the class `visible`.

It's cleaned up nicely.  Thank you for the comments.